Week3/WEEK 3 SOLUTION.ipynb

```diff
@@ -0,0 +1,931 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "b624b222",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sqlite3 as sq"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "id": "d3dcddd9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "con = sq.connect('ESM.db')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "id": "034d4853",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cur = con.cursor()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "44955bfa",
+   "metadata": {},
+   "source": [
+    "1.Create Table Employee."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "id": "c4ba7afa",
+   "metadata": {},
```

Comment:
what the meaning of "metadata"

Reply:
metadata is structured reference data that helps to sort and identify attributes of the information it describes

Reply:
If my answer is wrong, please tell me the current answer of this question.